/dxl_driver:
  ros__parameters:
    driver:
      frequency: 1.0

    servos:
      ids: [1] # Servo IDs
      operating_modes: [4] # Driver operating mode, [4 is position control]
      homing_modes: [0] # Homing mode, [0 is zero position at startup]
      directions: [1] # Direction of rotation for each servo [-1, 1]
      max_speeds: [1.2] # Maximum allowable speed in rad/s
      max_currents: [100.] # Maximum allowable current in mA [ENFORCEMENT NOT IMPLEMENTED]
      gear_ratios: [3.0] # Gear ratio between servo horn and output shaft
